google/cloud/pubsublite/internal/wire/single_partition_publisher.py

```diff
@@ -54,17 +57,27 @@
 _MAX_MESSAGES = 1000
 
 
+class _SequencedMessage(NamedTuple):
```

Comment:
can you please call this a name that isn't already overloaded? MessageWithSequence?

Reply:
Renamed to `_MessageWithSequence`.